apps/meteor/app/lib/server/lib/notifyUsersOnMessage.ts

```diff
@@ -184,8 +163,7 @@ export async function notifyUsersOnMessage(message, room) {
 	}
 
 	// Update all the room activity tracker fields
-	await Rooms.incMsgCountAndSetLastMessageById(message.rid, 1, message.ts, settings.get('Store_Last_Message') && message);
-
+	await Rooms.incMsgCountAndSetLastMessageById(message.rid, 1, message.ts, settings.get('Store_Last_Message') ? (message as IMessage) : undefined);
```

Comment:
if you change the signature this can be like it was
```typescript
await Rooms.incMsgCountAndSetLastMessageById(message.rid, 1, message.ts, settings.get('Store_Last_Message') && message);
```

Reply:
Changed the signature as suggested. https://github.com/RocketChat/Rocket.Chat/pull/31877/commits/359e5dc9816d42e8cfe37721ecf1a7df76e9f27c

Reply:
https://github.com/RocketChat/Rocket.Chat/pull/31877/commits/5f39dfc565f70291947bda7d7badcd8130c26477